content/en/logs/log_collection/nodejs.md

```diff
@@ -128,6 +128,53 @@ logs:
     sourcecategory: sourcecode
 ```
 
+## Aggentless Logging
+
+It is possible to stream logs from your application to Datadog or to the Datadog Agent directly. This is not the recommended setup as handling connection issues should not be done directly in your application, but it might not be possible to log to a file when your application is running on a machine that cannot be accessed.
```

Comment:
Logging to a file is discouraged, so this paragraph doesn't reflect real usage. Maybe it should be clearer that the problem is with the agent and not with a file?

Reply:
I'm sorry but no, from what I understood so far, the recommended set up is to :
1. log to a file
2. instal and setup the agent to watch that file.

This paragraph is a copy/paste from the c# doc https://docs.datadoghq.com/logs/log_collection/csharp/?tab=serilog#agentless-logging

Reply:
This doc may also need to be changed, but maybe not since .NET is used by a lot of older entreprise users with legacy systems, so they can be expected to use outdated approaches. This is not true for Node.

The best practice for cloud-native architectures is to send logs to stdout/stderr and let the infrastructure forward it to the right place. This could be a file, syslog, a log shipper, a remote endpoint, but the app should not have this knowledge unless you are in an environment where this is not possible. This is even in the [twelve-factor app](https://12factor.net/logs).

In that sense, the statement is misleading since it focuses on a single outdated way to store logs.

Reply:
I see. 
Would this wording be better for you? 
`The recommended integration is to configure the Agent on your server in order to have a more robust integration (ex: handling connection issues). If you cannot instal the Datadog Agent, you can directly stream your log to Datadog.`

Reply:
That works! It's generic and to the point 👍